### PR TITLE
docs: shell syntax highlighting

### PR DESCRIPTION
**Pull request by emschwartz**

### CODE REVIEW

File: docs/quick-start/cli-repl.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ replicas and do some accounting!
 First let's create two accounts. (Don't worry about the details, you
 can read about them later.)
 
-```console
+```shell title="Start the REPL"
```
Comment (matklad) [inline docs/quick-start/cli-repl.md:13]: What's the "title" attriute for?

File: src/clients/dotnet/docs.zig [hunk 2]
```diff
@@ -25,7 +25,7 @@ pub const DotnetDocs = Docs{
     \\And if you do not already have NuGet.org as a package
     \\source, make sure to add it:
     \\
-    \\```console
+    \\```shell
```
Comment (matklad) [inline src/clients/dotnet/docs.zig:28]: I think you might need to update this for other languages as well?

**Review by matklad — APPROVED** [review]
Make sense to me!

These generally should be either

````

```console
$ some command --with-args
```

````

or

````

```shell
some command --with-args
```

````

The `console` version is perhaps more pedantically correct and looks better, _but_ it doesn't play as nice with with copy button

File: docs/quick-start/cli-repl.md (hunk 1 above)
Comment (emschwartz) [reply docs/quick-start/cli-repl.md:13]: It's supposed to add a title to the block, but I'm not sure if it actually works https://docusaurus.io/docs/markdown-features/code-blocks#code-title

**Comment by matklad** [comment]
@emschwartz do you want to push this over the finish line, or shell we close?

**Pull request closed without merging**